src/server/services/procedures/this-x-does-not-exist/this-x-does-not-exist.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Provides access to images from various websites typically called this<X>doesnotexist.com.
+ * These are images of X generated from random noise by (typically) a GAN.
+ * 
+ * @alpha
+ * @service
+ * @category Science
+ */
+'use strict';
+
+// I got the list of X to support from: https://thisxdoesnotexist.com/ and a few reddit pages.
+// most are supported, but some were not applicable or inaccessible for direct download.
+
+const logger = require('../utils/logger')('this-x-does-not-exist');
+const request = require('request');
```

Comment:
`request` is actually deprecated so we are trying to move away from it. Could you rewrite it using `axios`?

Reply:
Sure, new commit uses `axios` instead.